dev/scripts/create-mysql-container.sh

```diff
@@ -2,10 +2,11 @@
 
 
 docker run -d --name dev-surface-mysql \
-              -p3306:3306 \
+              -p33306:3306 \
+              --health-cmd='mysqladmin ping --silent' --health-interval='5s' \
               -e MYSQL_INNODB_BUFFER_POOL_SIZE=512M \
               -e MYSQL_DATABASE=surface \
-              mysql-dev:8.0.30-18
+              -e MYSQL_ALLOW_EMPTY_PASSWORD=yes \
+              mysql:8.0.30
 # Hack to wait a bit for MySQL - better if we do it with a check in the container, TODO later
```

Comment:
comment no longer applies after sleep is gone

Reply:
Right, nice catch.